storage/client/queue/README.md

```diff
@@ -108,22 +108,22 @@ The queue service do operations on the queues in the storage account and manage
 
 ### Queue Service Client
 
-The client performs the interactions with the Queue service, create or delete a queue, getting and setting Queue properties, list queues in account, and get queue statistics. An asynchronous, `QueueServiceAsyncClient`, and synchronous, `QueueClient`, client exists in the SDK allowing for selection of a client based on an application's use case.
+The client performs the interactions with the Queue service, create or delete a queue, getting and setting Queue properties, list queues in account, and get queue statistics. An asynchronous, `queueStorageAsyncClient`, and synchronous, `QueueClient`, client exists in the SDK allowing for selection of a client based on an application's use case.
 Once you have the value of the SASToken you can create the queue service client with `${accountName}`, `${SASToken}`.
 ```Java
-String queueServiceURL = String.format("https://%s.queue.core.windows.net", accountName)
-QueueServiceClient queueServiceClient = new QueueServiceClientBuilder().endpoint(queueURL).credential(SASToken).build();
+String queueStorageURL = String.format("https://%s.queue.core.windows.net", accountName)
+QueueStorageClient queueStorageClient = new QueueStorageClientBuilder().endpoint(queueURL).credential(SASToken).build();
 
-QueueClient newQueueServiceClient = queueServiceClient.createQueue("myqueue");
+QueueClient newQueueStorageClient = queueStorageClient.createQueue("myqueue");
```

Comment:
Rename `newQueueStorageClient` to `newQueueClient`, it's not a QueueStorageClient instance.

Reply:
Done.